virttest/utils_net.py

```diff
@@ -4145,3 +4145,21 @@ def get_netkvm_param_value(vm, param):
         return value
     finally:
         session.close()
+
+
+def get_host_ipv4_addr():
```

Comment:
Host maybe have multi nics,  I suggest to get IP address by  nic's MAC or interface name.  you can return default route device's IP address by default.

e,g. 

`
import netifaces as ni

def get_host_ip(interface, ip_version=ni.AF_INET)
      if interface is None:
          interface = ni.gateways()['default'][ip_version][1]
      iface = ni.ifaddresses(interface)
      return iface[ip_version][0]['addr']
`

Reply:
@xutian， Thanks for your review, updated.

Reply:
looks OK to me.